mangaki/mangaki/views.py

```diff
@@ -63,7 +66,9 @@
 
 UTA_ID = 14293
 
-GHIBLI_IDS = [2591, 8153, 2461, 53, 958, 30, 1563, 410, 60, 3315, 3177, 106]
+
+def compute_token(username):
```

Comment:
This should not be in `views.py`. Please put it somewhere in`mangaki.utils`, or create `mangaki.tokens`. In addition, this should really use `django.utils.crypto.salted_hmac`:

```python
def compute_token(username, salt):
    return salted_hmac(salt, username).hexdigest()
```